authors/apps/authentication/urls.py

```diff
@@ -1,12 +1,12 @@
-from django.conf.urls import url
 from django.urls import path
 
 from .views import (
     LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
 )
+
 app_name = "authentication"
 urlpatterns = [
-   path('user/', UserRetrieveUpdateAPIView.as_view()),
-   path('users/', RegistrationAPIView.as_view()),
-   path('users/login/', LoginAPIView.as_view()),
+    path('user/', UserRetrieveUpdateAPIView.as_view(), name="user-retrieve-update"),
+    path('users/', RegistrationAPIView.as_view(), name="user-register"),
```

Comment:
Since you have an `app_name`, you can just name the URLs like `register` without the user part then you would reference the `url-name` as `authentication:register`